File: 1550_Three_Consecutive_Odds.py
class Solution(object):
    def threeConsecutiveOdds(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        for i in range(2, len(arr)):
            if arr[i] % 2 and arr[i - 1] % 2 and arr[i - 2] % 2:
                return True
        return False

File: 1550_Three_Consecutive_Odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& q) {
        for(int a=2;a<q.size();a++)
            if(q[a]%2 && q[a-1]%2 && q[a-2]%2) return true;
        return false;
    }
};
